docs/advanced/MINAR_migration.md

```diff
@@ -60,28 +60,28 @@ If you don't like asynchronous programming, or if you wrote your code in asynchr
 
 ## Porting strategy 2: use the optional mbed OS 5 event loop
 
-To help ease porting MINAR applications, and to provide support for asynchronous style programming, mbed OS 5 provides an optional event loop. The main documentation for the event loop can be found [here](https://docs.mbed.com/docs/mbed-os-api-reference/en/5.3/APIs/tasks/events/). In short, the mbed OS 5 event loop implementation consists of an [EventQueue class](https://github.com/ARMmbed/mbed-os/blob/master/events/EventQueue.h) that implements the storage for the events and has a `dispatch` function. There are quite a few differences between MINAR and `EventQueue`:
+To help ease porting MINAR applications, and to provide support for asynchronous style programming, mbed OS 5 provides an optional event loop. The main documentation for the event loop can be found [here](https://docs.mbed.com/docs/mbed-os-api-reference/en/5.3/APIs/tasks/events/). In short, the mbed OS 5 event loop implementation consists of an [EventQueue class](https://github.com/ARMmbed/mbed-os/blob/master/events/EventQueue.h) that implements the storage for the events and has a `dispatch` function. There are differences between MINAR and `EventQueue`:
 
 - MINAR and the mbed OS 5 event loop have incompatible APIs.
-- Both MINAR and `EventQueue` work with *events* (objects that are placed in the event queue). However, the interface and implementations of events in MINAR and mbed OS 5 are different, and that's also true for the APIs that use them. Take a look at [the Callback class](https://github.com/ARMmbed/mbed-os/blob/master/platform/Callback.h), [the Event class](https://github.com/ARMmbed/mbed-os/blob/master/events/Event.h) and [the EventQueue class](https://github.com/ARMmbed/mbed-os/blob/master/events/EventQueue.h) for more details about the mbed OS 5 implementation.
+- Both MINAR and `EventQueue` work with *events* (objects that are placed in the event queue). However, the interface and implementations of events in MINAR and mbed OS 5 are different, and that's also true for the APIs that use them. Look at [the Callback class](https://github.com/ARMmbed/mbed-os/blob/master/platform/Callback.h), [the Event class](https://github.com/ARMmbed/mbed-os/blob/master/events/Event.h) and [the EventQueue class](https://github.com/ARMmbed/mbed-os/blob/master/events/EventQueue.h) for more details about the mbed OS 5 implementation.
```

Comment:
Can we check if https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/index.html has the relevant links? Or even the RTOS page?